.openpublishing.redirection.json

```diff
@@ -6,31 +6,31 @@
     },
     {
       "source_path_from_root": "/docs/storage/azure-storage.md",
-      "redirect_url": "/dotnet/aspire/storage/azure-storage-components"
+      "redirect_url": "/dotnet/aspire/storage/azure-storage-integrations"
     },
     {
       "source_path_from_root": "/docs/database/quickstart-sql-server.md",
-      "redirect_url": "/dotnet/aspire/database/sql-server-components"
+      "redirect_url": "/dotnet/aspire/database/sql-server-integrations"
```

Comment:
Is this one pluralized because there's the ADO.NET level and EF Core level integrations?

Reply:
There are two articles, one was the [original component/integration article](https://learn.microsoft.com/en-us/dotnet/aspire/database/sql-server-component?tabs=dotnet-cli) and the [plural alternative is the tutorial](https://learn.microsoft.com/en-us/dotnet/aspire/database/sql-server-components) that shows a working example with additional conceptual content and supporting integration code.